src/components/label/label.md.scss

```diff
@@ -48,6 +48,12 @@ $label-md-margin:                     $item-md-padding-top ($item-md-padding-rig
   transition: transform 150ms ease-in-out;
 }
 
+html[dir=rtl] {
```

Comment:
Preferably, use `[dir="rtl"]` instead, to give the option to use rtl on any html component